src/main/java/net/mcreator/ui/dialogs/workspace/GeneratorSelector.java

```diff
@@ -125,36 +126,8 @@ public static GeneratorConfiguration getGeneratorSelector(Window parent, @Nullab
 			genStats.add(new JEmptyBox(20, 20));
 
 			JPanel supportedElements = new JPanel(new GridLayout(-1, 6, 7, 3));
-
-			addStatsBar(L10N.t(covpfx + "achievements"), "achievements", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "biomes"), "biomes", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "blocksitems"), "blocksitems", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "tabs"), "tabs", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "item_properties"), "itemproperties", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "damage_sources"), "damagesources", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "def_biome_features"), "defaultfeatures", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "enchantments"), "enchantments", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "enchantment_types"), "enchantmenttypes", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "entities"), "entities", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "projectiles"), "projectiles", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "mobspawntypes"), "mobspawntypes", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "fluids"), "fluids", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "game_modes"), "gamemodes", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "game_rules"), "gamerules", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "generation_steps"), "generationsteps", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "map_colors"), "mapcolors", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "materials"), "materials", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "particles"), "particles", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "path_node_types"), "pathnodetypes", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "effects"), "effects", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "potions"), "potions", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "sounds"), "sounds", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "sound_categories"), "soundcategories", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "step_sounds"), "stepsounds", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "plant_types"), "planttypes", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "screens"), "screens", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "villager_professions"), "villagerprofessions", supportedElements, stats);
-			addStatsBar(L10N.t(covpfx + "item_types"), "itemtypes", supportedElements, stats);
+			DataListLoader.getCache().entrySet().stream().filter(e -> !e.getValue().isEmpty()).map(Map.Entry::getKey)

```

Comment:
Certain data lists are not shown in the generator stats, such as "types", "features", ..., but after this PR, they are shown

Reply:
As far as I know, these lists are present in the cache too so value is not empty

Reply:
Hmm, I tried logging them and for types, features and directions I was getting `{}` (thought to that after seeing their bars were empty for the latest generator).

Reply:
Well, empty lists _should_ be shown in either case so logic is not changed. Even if you wanted to change that, you should in this case provide before-after screenshots and include this in the changelog

Reply:
I needed to sync RB changes before I could.

Reply:
No worries, I just wanted to clarify, not saying this should be fixed now/immediately hehe

Reply:
![image](https://github.com/MCreator/MCreator/assets/71347607/2b59409d-4b10-4be8-a3f6-216ee08e6c0a)
I changed code to remember what data lists were found as files (in the core plugin).